src/fileHandler/fileHandler.py

```diff
@@ -2,35 +2,63 @@
 import os
 
 
-def read(filename, line=False):  # pulls asingle line from a file
+def read(filePath, line=False):  # pulls a single line from a file
     try:
-        filename = open(filename, "r")
-        text = filename.readlines()
-        filename.close()
+        readFile = open(filePath, "r")
+        lines = readFile.readlines()
+        readFile.close()
         if not line:
-            line = text[line].rstrip()
+            line = lines[line].rstrip()
             return line
         else:
-            return text
+            return lines
     except Exception as e:
-        print('ERROR fileHandler.py read(filename,line)\n      ', e)
+        print('ERROR fileHandler.py read(filePath,line)\n      ', e)
 
 
-def readjson(filename):
+def readjson(filePath):
     if True:
-        filename = open(filename, "r")
-        output = filename.readlines()
-        text = ""
-        for i in range(0, len(output)):
-            text = text + output[i].replace("\n", "")
-        filename.close()
-        return text
+        jsonFile = open(filePath, "r")
+        lines = jsonFile.readlines()
+        json = ""
+        for i in range(len(lines)):
+            json = json + lines[i].replace("\n", "")
+        jsonFile.close()
+        return json
     try:
         pass
     except Exception as e:
         print('ERROR fileHandler.py readjson(filename,line)\n      ', e)
 
 
+<<<<<<< HEAD
```

Comment:
What is this?

Reply:
The File in general is just renaming to make it understandable.